Index: src/com/revature/app/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/revature/app/Main.java b/src/com/revature/app/Main.java
new file mode 100644
--- /dev/null	(date 1644435578388)
+++ b/src/com/revature/app/Main.java	(date 1644435578388)
@@ -0,0 +1,4 @@
+package com.revature.app;
+
+public class Main {
+}
Index: src/com/revature/app/Solution.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/com/revature/app/Solution.java b/src/com/revature/app/Solution.java
new file mode 100644
--- /dev/null	(date 1644435627141)
+++ b/src/com/revature/app/Solution.java	(date 1644435627141)
@@ -0,0 +1,8 @@
+package com.revature.app;
+
+public class Solution {
+    public boolean recurringString(String s) {
+
+        return false;
+    }
+}
